### PR TITLE
Jacoco code coverage integration

### PR DESCRIPTION
**Pull request by jahid1209**
- Added `Jacoco` code coverage.

- Changed `groupId` and `version` as per the discussion.

- Excluded `ScannerGlobalConfig` and `SecurityScanStep` class from code coverage. Now the code coverage is **70%**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,6 +122,36 @@
...
+            <phase>test</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <excludes>
+            <exclude>com/synopsys/integration/jenkins/scan/extension/**/*.class</exclude>
+            <exclude>com/synopsys/integration/jenkins/scan/SecurityScanner.class</exclude>
```
Comment (jahid1209) [inline pom.xml:148]: @kishorikumar @dmamidisynopsys and @devmehtasynopsys 
Intentionally excluded these **Jenkins API** specific classes from the code coverage.
 Let me know your thoughts!

Comment (dmamidisynopsys) [reply pom.xml:148]: Please test this locally and through pipeline before merging. Looks good overall

Comment (devmehtasynopsys) [reply pom.xml:148]: I am testing on pipeline and will edit if necessary

Comment (dmamidisynopsys) [reply pom.xml:149]: @jahid1209 @kishorikumar @devmehtasynopsys Lets not merge this change yet, I see new requirements (few additional) from Jenkins Infra team

**Review by dmamidisynopsys — COMMENTED** [review]
Added comment in main thread

**Pull request merged**